client/src/containers/searchresults/displaytable.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
```

Comment:
I put this at the top so you'd see it first. The more we can abstract logic from the return function the better. You have all the pieces there and the map function is exactly what we want to do but here are my suggestions for this file as you have it now:
```
import React, { Component } from 'react'
import { bindActionCreators } from 'redux'
import { connect } from 'react-redux'
import {
  Table,
  TableBody,
  TableRow,
  TableRowColumn,
  Avatar,
  FlatButton,
} from 'material-ui'

class DisplayTable extends Component {
  constructor(props) {
    super(props)

    this.renderTableRows = this.renderTableRows.bind(this)
  }

  renderTableRows() {
    const { users } = this.props

    const tableRows = users.map((user) => {
      const {
        id,
        first_name,
        last_name,
        org_name,
        linkedin_id,
        contact_url
      } = user

      const isStriped = (index) => {
        if (index % 2) return '#8195b1'
        return 'white'
      }
      const isStripedButton = (index) => {
        if (!(index % 2))
        return '#f8f8f8'
      }

      return (
        <TableRow style={{backgroundColor: isStriped(id)}} key={id}>
          <TableRowColumn className='table-cell'>
            <div className='table-name' style={{borderBottomColor: isStriped(id+1)}}>
              <h2>{first_name} {last_name}</h2>
            </div>
            <div className='table-icon' style={{borderRightColor: isStriped(id+1)}}>
              <div style={{paddingLeft: '12px'}}>
                <Avatar size={60} src={require('./portrait.png')} />
              </div>
              <FlatButton hoverColor={isStripedButton(id+1)} style={{float: 'left'}}>View profile</FlatButton>
            </div>
            <div className='table-about'>
              <ul>
                <li>Works at: {org_name}</li>
              </ul>
            </div>
            <ul className='table-contact'>
              <li>LinkedIn: {linkedin_id}</li>
              <li>Contact: {contact_url}</li>
            </ul>
          </TableRowColumn>
        </TableRow>
      )
    })

    return tableRows
  }

  render() {
    return (
      <Table selectable={false}>
        <TableBody displayRowCheckbox={false} stripedRows>
          {this.renderTableRows()}
        </TableBody>
      </Table>
    )
  }
}

export default connect(
)(DisplayTable)

```